bucket/endless-sky.json

```
@@ -1,31 +1,34 @@
 {
-    "description": "2D space trading and combat game similar to the classic Escape Velocity series",
+    "description": "2D space trading and combat game similar to the classic Escape Velocity series. It's also on Steam.",
```

The fact that it's on Steam isn't relevant for scoop-games users:

```suggestion
    "description": "2D space trading and combat game similar to the classic Escape Velocity series.",
```